mardi_importer/mardi_importer/cran/RPackage.py

```
@@ -221,18 +223,19 @@ def write(self) -> Optional[Dict[str, str]]:
         # Logic to determine if 'Author' is the class or the module containing the class
-        if hasattr(Author, 'Author') and not isinstance(Author, type):
+        if hasattr(Author, "Author") and not isinstance(Author, type):
             author_factory = Author.Author
         else:
             author_factory = Author
 
         if not callable(author_factory):
-            raise TypeError(f"Could not resolve a callable Author class. Check your imports.")
+            raise TypeError(
+                f"Could not resolve a callable Author class. Check your imports."
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)